platform/security/pdp/security-pdp-xacmlrealm/src/main/java/ddf/security/pdp/xacml/processor/BalanaPdp.java

```diff
@@ -330,28 +318,29 @@ public void endElement(String uri, String localName, String qName)
     /**
      * Marshalls the XACML request to a string.
      *
-     * @param xacmlRequestType
-     *            The XACML request to marshal.
+     * @param xacmlRequestType The XACML request to marshal.
      * @return A string representation of the XACML request.
      */
     private String marshal(RequestType xacmlRequestType) throws PdpException {
-        ObjectFactory objectFactory = new ObjectFactory();
-        Writer writer = new StringWriter();
-        JAXBElement<RequestType> xacmlRequestTypeElement = objectFactory
-                .createRequest(xacmlRequestType);
-
-        Marshaller marshaller = null;
         String xacmlRequest = null;
-
+        if (null == parser) {
```

Comment:
I'm not sure i understand you comment.  Java best practices puts null on the left as it cannot be assigned should you forget to use 2 equals.